client/scripts/views/components/inline_thread_composer.ts

```diff
@@ -96,8 +95,12 @@ const LinkPost: m.Component<ILinkPostAttrs, ILinkPostState> = {
         onkeyboardSubmit: createLink,
       }),
       m(AutoCompleteTagForm, {
```

Comment:
Looks like we can reuse a component such as this as its used more than once

Reply:
Yeah unfortunately they handle the state of their wrapping components, so it seems like slightly minimizing redundancy/saving a few lines of code would come at the cost of increased complexity?

Reply:
Creating a component called InlineThreadComposerTagForm that accepts `parentState` in its attrs seems pretty straightforward. I agree with you that it's not a clear cut tradeoff in general though